Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -789,7 +791,7 @@ void FinishLoadSceneHost()
 
             if (client.connection != null)
             {
-                OnClientConnect(client.connection);
+                client.OnAuthenticated(client.connection);
```

Comment:
I am very confused about what FinishLoadSceneHost is supposed to do.  Why do we set the connection to null here?

Reply:
From what I can tell it is nulled during a full scene change. None of the examples seem to cover this.